src/components/GalleryArrows/Arrow.tsx

```diff
@@ -0,0 +1,12 @@
+import { FC } from "react";
+import { ArrowStyled } from "./Arrows.styled";
+
+export interface ArrowProps {
+  onClick?: () => void;
+  $direction: "prev" | "next";
+}
+
+export const Arrow: FC<ArrowProps> = (props) => {
+  const { onClick, $direction } = props;
```

Comment:
One suggestion -`$` is used in Styled Components props due to a conflict in prop naming and CSS styling - you can abstract it in the component, by e.g. naming it just `direction` and then just passing `$direction={direction}` to the styled component below. This way you won't expose the internal styled component-specific implementation to the outside world and keep prop names consistent.